lib/ibm_watson/visual_recognition_v3.rb

```diff
@@ -237,8 +237,7 @@ def detect_faces(images_file: nil, url: nil, images_file_content_type: nil, imag
     #   file names, and classifier and class names). The service assumes UTF-8 encoding if
     #   it encounters non-ASCII characters.
     # @param name [String] The name of the new classifier. Encode special characters in UTF-8.
-    # @param args [Hash] Hash for optional parameters
-    # @option args classname_positive_examples [File] A .zip file of images that depict the visual subject of a class in the new
+    # @param positive_examples [File] A .zip file of images that depict the visual subject of a class in the new
```

Comment:
I think we want to document `postive_examples` and `positive_examples_filename` (below) as a `Hash` with an appropriate description.